Library/Homebrew/formula_support.rb

```diff
@@ -28,13 +28,14 @@ def valid?
 
   def to_s
     return @explanation unless @explanation.empty?
+    os_name = OS.mac? ? MacOS.os_name : 'macOS'
```

Comment:
Is this what it's supposed to be doing? If `OS.mac?` is `false`, how can it be `'macOS'`?